docs/navigate.md

```diff
@@ -0,0 +1,82 @@
+## jquery.flot.navigate.js
```

Comment:
I think all these navigation related plugins should have a link to the Interactions.md file which provides a high level overview.